codespell_lib/data/dictionary.txt

```diff
@@ -21389,6 +21389,8 @@ meeds->needs
 meens->means
 meerkrat->meerkat
 meerly->merely
+meethod->method
```

Comment:
maybe:

meethodology->methodology

Reply:
Found a few hits in GitHub indeed. Done.